java-checks/src/test/java/org/sonar/java/checks/spring/TransactionalMethodVisibilityCheckTest.java

```diff
@@ -25,13 +28,35 @@
 class TransactionalMethodVisibilityCheckTest {
 
   @Test
-  void test() {
+  void test_Spring5() {
     CheckVerifier.newVerifier()
-      .onFile(mainCodeSourcesPath("checks/spring/TransactionalMethodVisibilityCheckSample.java"))
+      .onFile(mainCodeSourcesPath("checks/spring/TransactionalMethodVisibilityCheckSample_Spring5.java"))
       .withCheck(new TransactionalMethodVisibilityCheck())
       .verifyIssues();
   }
 
+  @Test
+  void test_spring6() {
+    CheckVerifier.newVerifier()
+      .onFile(mainCodeSourcesPath("checks/spring/TransactionalMethodVisibilityCheckSample_Spring6.java"))
+      .withCheck(new TransactionalMethodVisibilityCheck())
+      .withClassPath(List.of(new File("spring-tx-6.0.1.jar"),
+        new File("spring-context-6.0.2.jar")))
```

Comment:
Why do we test with 2 versions of Spring 6?

Reply:
No particular reason. I will use the same for both to avoid confusion.